lerobot/scripts/eval.py

```diff
@@ -546,7 +543,7 @@ def eval(
             policy,
             hydra_cfg.eval.n_episodes,
             max_episodes_rendered=10,
-            video_dir=Path(out_dir) / "eval",
+            video_dir=Path(out_dir) / "video",
```

Comment:
I am ok with this, but it would be awesome to add a test for it in this PR. Not a blocker for merge. Thanks!

Reply:
Eh, it's the end of the day... I'm gonna except your no-block :P